mappings/com/mojang/blaze3d/platform/GlStateManager.mapping

```diff
@@ -216,61 +291,77 @@ CLASS com/mojang/blaze3d/platform/GlStateManager
 		ARG 6 format
 		ARG 7 type
 		ARG 8 pixels
-	METHOD _vertexAttribIPointer (IIIIJ)V
+	METHOD _vertexAttribIPointer _vertexAttribIPointer (IIIIJ)V
 		ARG 0 index
 		ARG 1 size
 		ARG 2 type
 		ARG 3 stride
 		ARG 4 pointer
-	METHOD _vertexAttribPointer (IIIZIJ)V
+	METHOD _vertexAttribPointer _vertexAttribPointer (IIIZIJ)V
 		ARG 0 index
 		ARG 1 size
 		ARG 2 type
 		ARG 3 normalized
 		ARG 4 stride
 		ARG 5 pointer
-	METHOD _viewport (IIII)V
+	METHOD _viewport _viewport (IIII)V
 		ARG 0 x
 		ARG 1 y
 		ARG 2 width
 		ARG 3 height
-	METHOD glActiveTexture (I)V
+	METHOD getBoundFramebuffer getBoundFramebuffer ()I
+	METHOD glActiveTexture glActiveTexture (I)V
 		ARG 0 texture
-	METHOD glAttachShader (II)V
+	METHOD glAttachShader glAttachShader (II)V
 		ARG 0 program
 		ARG 1 shader
-	METHOD glBlendFuncSeparate (IIII)V
+	METHOD glBlendFuncSeparate glBlendFuncSeparate (IIII)V
 		ARG 0 srcFactorRGB
 		ARG 1 dstFactorRGB
 		ARG 2 srcFactorAlpha
 		ARG 3 dstFactorAlpha
-	METHOD glCheckFramebufferStatus (I)I
+	METHOD glCheckFramebufferStatus glCheckFramebufferStatus (I)I
 		ARG 0 target
-	METHOD glCompileShader (I)V
+	METHOD glCompileShader glCompileShader (I)V
 		ARG 0 shader
-	METHOD glCreateShader (I)I
+	METHOD glCreateProgram glCreateProgram ()I
+	METHOD glCreateShader glCreateShader (I)I
 		ARG 0 type
-	METHOD glDeleteProgram (I)V
+	METHOD glDeleteProgram glDeleteProgram (I)V
 		ARG 0 program
-	METHOD glDeleteShader (I)V
+	METHOD glDeleteShader glDeleteShader (I)V
 		ARG 0 shader
-	METHOD glGetProgramInfoLog (II)Ljava/lang/String;
+	METHOD glGenFramebuffers glGenFramebuffers ()I
+	METHOD glGenRenderbuffers glGenRenderbuffers ()I
+	METHOD glGetProgramInfoLog glGetProgramInfoLog (II)Ljava/lang/String;
 		ARG 0 program
 		ARG 1 maxLength
-	METHOD glGetProgrami (II)I
+	METHOD glGetProgrami glGetProgrami (II)I
 		ARG 0 program
 		ARG 1 pname
-	METHOD glGetShaderInfoLog (II)Ljava/lang/String;
+	METHOD glGetShaderInfoLog glGetShaderInfoLog (II)Ljava/lang/String;
 		ARG 0 shader
 		ARG 1 maxLength
-	METHOD glGetShaderi (II)I
+	METHOD glGetShaderi glGetShaderi (II)I
 		ARG 0 shader
 		ARG 1 pname
-	METHOD glLinkProgram (I)V
+	METHOD glLinkProgram glLinkProgram (I)V
 		ARG 0 program
-	METHOD glShaderSource (ILjava/util/List;)V
+	METHOD glShaderSource glShaderSource (ILjava/util/List;)V
 		ARG 0 shader
 		ARG 1 strings
+	METHOD lambda$static$0 (I)Lcom/mojang/blaze3d/platform/GlStateManager$C_vgejkzlz;
+		ARG 0 id
+	METHOD setupGui3DDiffuseLighting setupGui3DDiffuseLighting (Lnet/minecraft/unmapped/C_zviwfwgf;Lnet/minecraft/unmapped/C_zviwfwgf;)V
+		ARG 0 light0
+		ARG 1 light1
+	METHOD setupGuiFlatDiffuseLighting setupGuiFlatDiffuseLighting (Lnet/minecraft/unmapped/C_zviwfwgf;Lnet/minecraft/unmapped/C_zviwfwgf;)V
+		ARG 0 light0
+		ARG 1 light1
+	METHOD setupLevelDiffuseLighting setupLevelDiffuseLighting (Lnet/minecraft/unmapped/C_zviwfwgf;Lnet/minecraft/unmapped/C_zviwfwgf;Lnet/minecraft/unmapped/C_gxvddjib;)V
```

Comment:
```suggestion
	METHOD setupLevelDiffuseLighting setupWorldDiffuseLighting (Lnet/minecraft/unmapped/C_zviwfwgf;Lnet/minecraft/unmapped/C_zviwfwgf;Lnet/minecraft/unmapped/C_gxvddjib;)V
```
These three `diffuseLighting` names in general seem clunky, but I'm not entirely sure what better names might be

Reply:
Basically these 3 methods use some constants to set some shader context. It makes sense when looking at the code